install.sh

```diff
@@ -167,6 +167,98 @@ mkdir -p "$OMS_CACHE/runtime-script"
 mkdir -p "$OMS_CACHE/startup-script"
 mkdir -p "$OMS_CACHE/trash"
 
+# install lolcat with apt-get if not found
+if [ ! -f "$OMS/lib/bin/lolcat" ]
+then
+  _info '  Installing lolcat...'
+  # install lolcat with apt-get if operating system is debian/ubuntu or based on debian/ubuntu
+  if [ -e "/etc/debian_version" ]
+  then
+    # check if is ubuntu
+    if [ -e "/etc/lsb-release" ]
+    then
+      if grep -q "Ubuntu" "/etc/lsb-release"
+      then
+        # install lolcat with apt-get if snap is not installed or if snapd not running
+        # so if snap is installed and snapd is running, we will use snap to install lolcat 
+        if [[ -e "/usr/bin/snap " ]] && [[ -f "/usr/bin/snap" ]] && [[ -e "/usr/lib/snapd" ]] && [[ -d "/usr/lib/snapd"]]
+        then
+          #later ...an idea: verify if systemd have snapd running
+          _info ' Snap is installed, you can installing lolcat with snap !'
+          # ask if user want to install lolcat with snap
+
+          echo -n "Proceed? [y/n]: "
+          read answer
+          if [[ $answer == "y" ]] || [[ $answer == "Y" ]] || [[ $answer == "yes" ]] || [[ $answer == "Yes" ]]
+          then
+            # install lolcat with snap if the user want to use snap
+            _info '    Installing lolcat with snap'
+            snap install lolcat-c
+          else
+            _info '  Skipping lolcat installation'
+            _info '  Intalling lolcat with apt-get'
+            apt-get install -y lolcat
+          fi
+        # else lolcat install with apt-get
+        else
+          apt-get install -y lolcat
+        fi
+      fi
+    else
+      apt-get install -y lolcat
+    fi
+  fi
+  # install lolcat with yum if operating system is centos/redhat/fedora or based on centos/redhat/fedora
+  if [ -e "/etc/redhat-release" ]
+  then
+    yum install -y lolcat
+  fi
+  # install lolcat with AUR if operating system is archlinux
+  if [ -e "/etc/arch-release" ]
+  then
+    git clone https://aur.archlinux.org/packages/c-lolcat
```

Comment:
For Arch users, they prefer `pacman` to AUR.

Reply:
okay